spec/system/candidate_interface/candidate_apply_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe 'A candidate applying from Find' do
+  let(:provider_code) { '1AB' }
```

Comment:
Non-blocking: I'd put these values either directly in the `it` blocks, or use a constant. Lets don't add that much imho.

Reply:
what about using random values, maybe using `Faker`?

Reply:
@tijmenb agree with not using `let` in this case

@alexdesi hmm, we could, but similar to `let`, is there a need for it?

Reply:
Lol:

```ruby
RSpec/LeakyConstantDeclaration: Stub constant instead of declaring explicitly. (http://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/LeakyConstantDeclaration)
  PROVIDER_CODE = '1AB'
```

I'm leaving the `let`s in for now...